lib/heart_seed/db_seed.rb

```diff
@@ -17,30 +17,49 @@ def self.bulk_insert(source_file: nil, model_class: nil)
       end
     end
 
+    # delete all records and insert from seed yaml
+    #
+    # @param file_path [String]
+    # @param model_class [Class] require. extends {ActiveRecord::Base}
+    def self.insert(file_path: nil, model_class: nil)
+      fixtures = HeartSeed::Converter.read_fixture_yml(file_path)
+      model_class.transaction do
+        model_class.delete_all
+        fixtures.each do |fixture|
+          model_class.create(fixture)
+        end
+      end
+    end
+
     # import all seed yaml to table
     #
-    # @param seed_dir [String]
-    # @param tables   [Array<String>,String] table names array or comma separated table names. if empty, import all seed yaml. if not empty, import only these tables.
-    # @param catalogs [Array<String>,String] catalogs names array or comma separated catalog names. if empty, import all seed yaml. if not empty, import only these tables in catalogs.
-    def self.import_all(seed_dir: HeartSeed::Helper.seed_dir, tables: ENV["TABLES"], catalogs: ENV["CATALOGS"])
+    # @param seed_dir    [String]
+    # @param tables      [Array<String>,String] table names array or comma separated table names. if empty, import all seed yaml. if not empty, import only these tables.
+    # @param catalogs    [Array<String>,String] catalogs names array or comma separated catalog names. if empty, import all seed yaml. if not empty, import only these tables in catalogs.
+    # @param insert_mode [String] 'bulk' or other string. if empty or 'bulk', using bulk insert. if not empty and not 'bulk', import with ActiveRecord.
+    def self.import_all(seed_dir: HeartSeed::Helper.seed_dir, tables: ENV["TABLES"], catalogs: ENV["CATALOGS"], insert_mode: ENV["INSERT_MODE"] || "bulk")
```

Comment:
Line is too long. [154/130]<br>Perceived complexity for import_all is too high. [9/7]